addons/binding/org.openhab.binding.honeywellwifithermostat/src/main/java/org/openhab/binding/honeywellwifithermostat/handler/honeywellWifiThermostatHandler.java

```diff
@@ -0,0 +1,172 @@
+/**
+ * Copyright (c) 2014 openHAB UG (haftungsbeschraenkt) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.honeywellwifithermostat.handler;
+
+import static org.openhab.binding.honeywellwifithermostat.honeywellWifiThermostatBindingConstants.*;
+
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+
+import org.eclipse.smarthome.config.core.Configuration;
+import org.eclipse.smarthome.core.library.types.DecimalType;
+import org.eclipse.smarthome.core.library.types.OnOffType;
+import org.eclipse.smarthome.core.library.types.StringType;
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.ThingStatus;
+import org.eclipse.smarthome.core.thing.ThingStatusDetail;
+import org.eclipse.smarthome.core.thing.binding.BaseThingHandler;
+import org.eclipse.smarthome.core.types.Command;
+import org.openhab.binding.honeywellwifithermostat.internal.data.HoneywellThermostatData;
+import org.openhab.binding.honeywellwifithermostat.internal.data.HoneywellThermostatFanMode;
+import org.openhab.binding.honeywellwifithermostat.internal.data.HoneywellThermostatSystemMode;
+import org.openhab.binding.honeywellwifithermostat.internal.webapi.HoneywellWebsiteJetty;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link honeywellWifiThermostatHandler} is responsible for handling commands, which are
+ * sent to one of the channels.
+ *
+ * @author JD Steffen - Initial contribution
+ */
+public class honeywellWifiThermostatHandler extends BaseThingHandler {
```

Comment:
It seems you have used the archetype with a lowercase parameter for creation. This now results in all your classes starting lower case, which is not correct.
Please refactor them to start with a capital letter, i.e. here "HoneywellWifiThermostatHander".